src/litepicker.ts

```diff
@@ -333,7 +344,8 @@ export class Litepicker extends Calendar {
 
       if (this.options.splitView) {
         const monthItem = target.closest(`.${style.monthItem}`);
-        idx = [...monthItem.parentNode.childNodes].findIndex(el => el === monthItem);
```

Comment:
What is wrong with `findIndex` ? Why you are create a custom function ?

Reply:
spread operator used here is not good.

Reply:
Could you explain why ?